InnerEye/ML/Histopathology/datamodules/tcga_crck_module.py

```diff
@@ -0,0 +1,33 @@
+from typing import Tuple, Any
+
+from InnerEye.ML.Histopathology.datamodules.base_module import TilesDataModule
+from InnerEye.ML.Histopathology.datasets.tcga_crck_tiles_dataset import TcgaCrck_TilesDataset
+from InnerEye.ML.utils.split_dataset import DatasetSplits
```

Comment:
not sure if we want to do it in this set of PRs but we said DatasetSplits could be moved to hi-ml as well

Reply:
Yes it can move at some point - I felt that since it is already OS it is not urgent to move it right now